### PR TITLE
Fix UI Error Handling on pipeline entity page

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
See #3380 
### Describe your changes :

I worked on UI Error Handling for pipeline entity page

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t, @darth-coder00 @vivekratnavel  @ShaileshParmar11

### CODE REVIEW

**Review by vivekratnavel — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/main/resources/ui/src/utils/StringsUtils.ts
Comment (darth-coder00) [inline openmetadata-ui/src/main/resources/ui/src/utils/StringsUtils.ts:105]:
```suggestion
export const getErrorText = (value: AxiosError | string, fallbackText: string) => {
  return (isString(value) ? value : value.response?.data?.message) || fallbackText;
};
```

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
One change suggested

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
@Sachin-chaurasiya LGTM!, Let's add some unit tests to support these error handling changes before we merge it in

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya LGTM!, Let's add some unit tests to support these error handling changes before we merge it in

@darth-coder00 , we can merge this PR and In my next patch I will add the unit tests to support these error handling.

**Review by darth-coder00 — APPROVED** [review]
As discussed, Unit tests to be added in next patch

**Comment by darth-coder00** [comment]
@vivekratnavel @harshach please can you review and merge this if looking good.

**Review by vivekratnavel — APPROVED** [review]

**Comment by vivekratnavel** [comment]
@harshach Can you merge this?

**Pull request merged**
